docs/data/base/guides/working-with-tailwind-css/PlayerFinal.js

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+
+export default function PlayerFinal() {
+  return (
+    <iframe
```

Comment:
Current master is broken, this is the fix from this branch: https://codesandbox.io/s/github/mnajdova/material-ui/tree/docs/improve-base-with-taiwlind-css-guide/examples/mui-base-with-tailwind-css